Rework Quick Start to include benefits, remote execution, and core concepts (#11461)
PrefectHQ/prefect: docs/getting-started/quickstart.md

<div class="terminal">

```bash
python my_workflow.py
```

</div>

Now when we run this script, Prefect will automatically track the state of the flow run and log the output where we can see it in the UI or CLI.

<div class="terminal">

```bash
15:16:34.705 | INFO    | prefect.engine - Created flow run 'frisky-reindeer' for flow 'get-repo-info'
15:16:34.706 | INFO    | Flow run 'frisky-reindeer' - View at https://app.prefect.cloud/account/abc/flow-runs/flow-run/123
15:16:35.053 | INFO    | Flow run 'frisky-reindeer' - PrefectHQ/prefect repository statistics 🤓:
15:16:35.055 | INFO    | Flow run 'frisky-reindeer' - Stars 🌠 : 13593
15:16:35.744 | INFO    | Flow run 'frisky-reindeer' - Finished in state Completed()
```

</div>

You should see similar output in your terminal, with your own randomly generated flow run name and your own Prefect Cloud account URL.

## Step 4: Choose a remote infrastructure location

Let's get this workflow running on infrastructure other than your local machine!
We can tell Prefect where we want to run our workflow by creating a [work pool](/concepts/work-pools/).

> Because we're using Prefect Cloud, we have access to Prefect Managed work pools that provide hosted infrastructure for running our flows.

Reviewer: ```suggestion
We can have Prefect Cloud run our flow code for us with a Prefect Managed work pool.
```

description: Get started with Prefect, the easiest way to orchestrate and observe your data pipelines
tags:
    - getting started
    - quickstart
    - overview
search:
  boost: 2
---

# Quickstart

> Prefect is an orchestration and observability platform that empowers developers to build and scale resilient code quickly, turning their scheduled jobs into resilient, recurring workflows.

Reviewer: ```suggestion
Prefect is an orchestration and observability platform that empowers developers to build and scale resilient code quickly, turning their python scripts into resilient, recurring workflows.
```

In this quickstart, you'll see how you can schedule your code on remote infrastructure and observe the state of your workflows.
With Prefect, you can go from a Python script to a production-ready workflow that runs remotely in minutes.

Let's get started!

## Setup

> Here's a basic script so we have something to work with.

Reviewer: ```suggestion
```

> The script below fetches statistics about the [main Prefect GitHub repository](https://github.com/PrefectHQ/prefect).

Reviewer: ```suggestion
Here's a basic script that fetches statistics about the [main Prefect GitHub repository](https://github.com/PrefectHQ/prefect).
```

<div class="terminal">

```bash
python my_workflow.py
```

</div>

Now when we run this script, Prefect will automatically track the state of the flow run and log the output where we can see it in the UI or CLI.

<div class="terminal">

```bash
15:16:34.705 | INFO    | prefect.engine - Created flow run 'frisky-reindeer' for flow 'get-repo-info'
15:16:34.706 | INFO    | Flow run 'frisky-reindeer' - View at https://app.prefect.cloud/account/abc/flow-runs/flow-run/123
15:16:35.053 | INFO    | Flow run 'frisky-reindeer' - PrefectHQ/prefect repository statistics 🤓:
15:16:35.055 | INFO    | Flow run 'frisky-reindeer' - Stars 🌠 : 13593
15:16:35.744 | INFO    | Flow run 'frisky-reindeer' - Finished in state Completed()
```

</div>

You should see similar output in your terminal, with your own randomly generated flow run name and your own Prefect Cloud account URL.

## Step 4: Choose a remote infrastructure location

Let's get this workflow running on infrastructure other than your local machine!
We can tell Prefect where we want to run our workflow by creating a [work pool](/concepts/work-pools/).

> Because we're using Prefect Cloud, we have access to Prefect Managed work pools that provides hosted infrastructure for running our flows.

Reviewer: ```suggestion
Because we're using Prefect Cloud, we have access to Prefect Managed work pools that provide hosted infrastructure for running our flows.
```
